trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/compute_broadband_noise.py

```diff
@@ -1,123 +1,349 @@
+        # equation 7
+        epsilon       = X**2 + (beta_sq)*(Y**2 + Z**2)
+        gamma         = np.sqrt(((mu/epsilon)**2)*(X**2 + beta_sq*(Z**2)))
+        ss_1, cc_1    = fresnel(2*((((omega/(1 +  (Omega*r/c_0)*(X/R_s))) /(0.8*U_inf)) /c) + (mu/c)*M + (gamma/c)))
+        ss_2, cc_2    = fresnel(2*((mu/c)*X/epsilon + (gamma/c)) )
+        triangle      = (omega/(U_inf*c)) - (mu/c)*X/epsilon + (mu/c)*M
+        norm_L_sq     = (1/triangle)*abs(np.exp(1j*2*triangle)*((1 - (1 + 1j)*(cc_1 - 1j*ss_1)) \
+                        + ((np.exp(-1j*2*triangle))*(np.sqrt((((omega/(1 +  (Omega*r/c_0)*(X/R_s))) /(0.8*U_inf)) + mu*M + gamma)/(mu*X/epsilon +gamma))) \
+                           *(1 + 1j)*(cc_2 - 1j*ss_2)) ))
+        norm_L_sq     = np.nan_to_num(norm_L_sq)
+
+        # ------------------------------------------------------------
+        # ****** EMPIRICAL WALL PRESSURE SPECTRUM ******
+        ones                = np.ones_like(Theta)
+        beta_c              = (Theta/tau_w)*dp_dx
+        d                   = 4.76*((1.4/(delta/delta_star))**0.75)*(0.375*(3.7 + 1.5*beta_c) - 1)
+        a                   = (2.82*((delta/delta_star)**2)*(np.power((6.13*((delta/delta_star)**(-0.75)) + d),(3.7 + 1.5*beta_c))))*\
+                              (4.2*((0.8*((beta_c + 0.5)**3/4))/(delta/delta_star)) + 1)
+        d_star              = d
+        d_star[beta_c<0.5]  = np.maximum(ones,1.5*d)[beta_c<0.5]
+        Phi_pp_expression   =  (np.maximum(a, (0.25*beta_c - 0.52)*a)*((omega*delta_star/Ue)**2))/(((4.76*((omega*delta_star/Ue)**0.75) \
+                               + d_star)**(3.7 + 1.5*beta_c))+ (np.power((8.8*(((delta/Ue)/(kine_visc/(((tau_w/rho)**0.5) **2)))**(-0.57))\
+                               *(omega*delta_star/Ue)),(np.minimum(3*ones,(0.139 + 3.1043*beta_c)) + 7)) ))
+        Phi_pp              = ((tau_w**2)*delta_star*Phi_pp_expression)/Ue
+        Phi_pp[np.isinf(Phi_pp)] = 0.
+        Phi_pp[np.isnan(Phi_pp)] = 0.
+
+        # ------------------------------------------------------------
+        # ****** DIRECTIVITY ****** 
+        X_2 = np.repeat(bspv.vehicle_coordinate_sys[:,:,:,:,:,:,0,:],2,axis = 6)
+        Y_2 = np.repeat(bspv.vehicle_coordinate_sys[:,:,:,:,:,:,1,:],2,axis = 6)
+        Z_2 = np.repeat(bspv.vehicle_coordinate_sys[:,:,:,:,:,:,2,:],2,axis = 6)        
+        A4    = bspv.M_hub_Y + Y_2 
+        A3    = bspv.cos_t_v_t_r*(bspv.cos_t_v*(bspv.M_hub_Z + Z_2) - bspv.sin_t_v*(bspv.M_hub_X + X_2)) \
+            - bspv.sin_t_v_t_r*(bspv.cos_t_v*(bspv.M_hub_X + X_2) + bspv.sin_t_v*bspv.M_hub_Z + Z_2) + r 
+        A2    =  bspv.cos_t_v_t_r*(bspv.cos_t_v*(bspv.M_hub_X + X_2) + bspv.sin_t_v*(bspv.M_hub_Z + Z_2))\
+            + bspv.sin_t_v_t_r*(bspv.cos_t_v*(bspv.M_hub_Z + Z_2) - bspv.sin_t_v*bspv.M_hub_X + X_2) - r*bspv.cos_phi
+        A1    = (bspv.cos_alpha_eff*A3 + bspv.sin_alpha_eff*A4)**2  
+
+        # Acousic Power Spectrial Density from each blade
```

Comment:
acoustic and spectral

Reply:
Should just be power spectral density. corrections made